### PR TITLE
Add LAN to events, move old workshops

### PR DESCRIPTION
**Pull request by LavenderGoons**
Don't have Tylor's slides can be added later

### CODE REVIEW

File: events.html [hunk 1]
```diff
@@ -52,7 +52,11 @@
         <div class="col-sm-7 float-right-md-lg">
             <h1>Upcoming Events</h1>
             <div class="event-list">
-
+                <a class="event" href="https://www.eventbrite.ca/e/frostycon-csc-game-night-7-tickets-31724885988">
+                    <h3>IBM Hackathon & CSC Lan Party #7<b>iTel</b></h3>
```
Comment (mitchhentges) [inline events.html]: Please make `&` -> `&amp;`

Comment (mitchhentges) [inline events.html]: Missing the `@`

File: events.html [hunk 2]
```diff
@@ -52,7 +52,11 @@
...
+                    <h5>February 11th - 9:00am to February 12th - 2:00am</h5>
```
Comment (mitchhentges) [inline events.html]: Follow the date format of previous events

File: events.html [hunk 3]
```diff
@@ -52,7 +52,11 @@
...
+                    Day Hackathon with IBM Watson/Bluemix from 9:00am to 5:00pm (lunch not provided). Boardgames and LAN setup 5:00 pm to 7:30pm, LAN Party 7:30pm to 2:00am, and Pizza at 8:30pm.
```
Comment (mitchhentges) [inline events.html]: Surely we can provide some better text here, explaining what the hackathon is about (more than just that it _has_ IBM Watson), and perhaps a little more on the LAN party. I wouldn't be explicit right here about lunch not being provided.

File: events.html [hunk 4]
```diff
@@ -62,24 +66,11 @@
             <h1>Previous Events</h1>
             <div class="event-list">
                 <div class="event">
-                    <h3>Advanced Topics of Java: Minecraft Modding Workshop @ <b>IB 1010</b></h3>
```
Comment (mitchhentges) [inline events.html]: Do these as part of a separate PR

File: events.html [hunk 5]
```diff
@@ -62,24 +66,11 @@
...
-                    <h5>6:00pm on Wednesday, February 1st, 2017</h5>
-                    Would you like a stronger understanding of Java? Do you enjoy playing games? Well now you can have both! Through this seminar you will learn how to use your Java skill to make a mod for Minecraft.
-                </div>
-                <div class="event">
                     <h3>Bowling night @ <b>Bowler Time</b></h3>
                     <h5>6:00pm on Friday, December 2nd, 2016</h5>
                     Put your Mountain Dew back in your cupboard and roll up your bag of Doritos, it's time to prove your skillz IRL in the most xtreme way possible: Bowling.
                 </div>
                 <div class="event">
-                    <h3>OSS License Workshop @ <b>IB 1010</b></h3>
```
Comment (mitchhentges) [inline events.html]: We shouldn't remove this until we have it in the "Previous Workshops" section

Comment (LavenderGoons) [inline events.html]: It is moved to the previous workshops section

Comment (mitchhentges) [inline events.html]: Ah, yeah, but it'll have less details than it does now, since it doesn't/can't link to the slides

File: events.html [hunk 6]
```diff
@@ -53,9 +53,10 @@
             <h1>Upcoming Events</h1>
             <div class="event-list">
                 <a class="event" href="https://www.eventbrite.ca/e/frostycon-csc-game-night-7-tickets-31724885988">
-                    <h3>IBM Hackathon & CSC Lan Party #7<b>iTel</b></h3>
-                    <h5>February 11th - 9:00am to February 12th - 2:00am</h5>
-                    Day Hackathon with IBM Watson/Bluemix from 9:00am to 5:00pm (lunch not provided). Boardgames and LAN setup 5:00 pm to 7:30pm, LAN Party 7:30pm to 2:00am, and Pizza at 8:30pm.
+                    <h3>IBM Hackathon &amp; CSC Lan Party #7 @ <b>iTel</b></h3>
+                    <h5>9:00am Saturday, February 11th to 2:00am Sunday, February 12th</h5>
+                    Starting 9:00am on Saturday we will be hosting a all day Hackathon with IBM's Watson. First we will be creating interfaces with Watson using their Bluemix API with Node.js and Android studio.
```
Comment (mitchhentges) [inline events.html]: "...hosting a**n** all day **h**ackathon...".
"Then **,** teams of 1 to 4 will hack together a project before 4:00pm."
